internal/pkg/utils/time.go

```diff
@@ -0,0 +1,29 @@
+//
+// Copyright (C) 2023 IOTech Ltd
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package utils
+
+import (
+	"time"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/v3/clients/logger"
+)
+
+// CheckMinInterval parses the ISO 8601 time duration string to Duration type
+// and evaluates if the duration value is smaller than the suggested minimum duration
+func CheckMinInterval(value string, minDuration time.Duration, lc logger.LoggingClient) bool {
```

Comment:
Remove return value since it is never used by callers

Reply:
Done